src/views/AddExpenses.js

```diff
@@ -1,11 +1,18 @@
-import { useState } from 'react';
-import { convertEurosToCents } from '../utils/helpers';
+import { useState, useRef, useEffect } from 'react';
+import { convertEurosToCents, getUKFormattedDate } from '../utils/helpers';
 
-const AddExpenses = ({ addExpense }) => {
+const AddExpenses = ({ addExpense, setMonthToShow }) => {
   const [title, setTitle] = useState('');
   const [date, setDate] = useState('');
   const [amount, setAmount] = useState('');
 
+  const titleInput = useRef(null);
+
+  // focus on title input on mount
```

Comment:
I noticed we do not have an auto-focus on any field on mount! Small UX improvement

Reply:
nice!